pyscf/sftda/__init__.py

```diff
@@ -0,0 +1,24 @@
+from pyscf import scf
+from pyscf.lib import logger
+
+try:
+    from pyscf.dft import KohnShamDFT
+    from pyscf.sftda import uks_sf
+    from pyscf.sftda.uks_sf import TDUKS_SF
+except (ImportError, IOError):
+    pass
```

Comment:
I would remove the `try`/`except` here as if a module cannot be loaded and there is no alternative, then the code should just crash. If a module is needed only in specific cases, then just lazy load the module in the specific function.

Reply:
OK.